### ISSUE TITLE
on backbutton press always close the app

### ISSUE THREAD

**Issue by selvam920** [jonataslaw/getx#2874]
after updating flutter 3.13 in android when back button pressed the app close the app instead of going to back previous page

**Comment by Silence-v** [comment]
cy ，，，same confusion ，

**Comment by minhx3** [comment]
same issues, please fix. thank

**Comment by minhx3** [comment]
when i downgrade to 3.10.1 it work

**Comment by Sumit258000** [comment]
Even 3.10.6 will also work. Problem starts from 3.13.0 +

**Comment by lidary-byte** [comment]
Check whether ```android:enableOnBackInvokedCallback="true"``` is added to the AndroidManifest.xml

**Comment by minhx3** [comment]
> Even 3.10.6 will also work. Problem starts from 3.13.0 +

https://github.com/jonataslaw/getx/pull/2882#issuecomment-1691532942

**Comment by minhx3** [comment]
> Check whether `android:enableOnBackInvokedCallback="true"` is added to the AndroidManifest.xml

thank for support, you save me 02 days

**Comment by Sumit258000** [comment]
> Check whether `android:enableOnBackInvokedCallback="true"` is added to the AndroidManifest.xml

Yes its already added still app closes on back press in flutter 3.13.0 +

**Comment by Sumit258000** [comment]
With `get: ^4.6.6` and `flutter:3.13.1` still problem persist.

**Comment by lidary-byte** [comment]
> > Check whether  is added to the AndroidManifest.xml`android:enableOnBackInvokedCallback="true"`
> 
> Yes its already added still app closes on back press in flutter 3.13.0 +

delete it

**Comment by Sumit258000** [comment]
Thanks man for your answer. I was struggling for this from last 2 weeks. Thanks a ton!!

**Comment by Duncan-Waweru** [comment]
> Check whether `android:enableOnBackInvokedCallback="true"` is added to the AndroidManifest.xml and delete it.

This is indeed the solution. Issue should be closed

**Comment by Isaacodes-dev** [comment]
I am facing the same issue but my application still loosing state and application is still closing

**Comment by islom9797** [comment]
> I am facing the same issue but my application still loosing state and application is still closing

After delete this line(android:enableOnBackInvokedCallback="true") , , flutter clean and flutter pub get helped me
